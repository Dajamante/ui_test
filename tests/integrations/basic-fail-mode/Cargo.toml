[package]
name = "basic_fail_mode"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dev-dependencies]
ui_test = { path = "../../.."}

[[test]]
name = "ui_tests"
harness = false

[[test]]
name = "run_file"
harness = true
