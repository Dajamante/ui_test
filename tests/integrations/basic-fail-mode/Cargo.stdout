
running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished 


running 2 tests
test run_file ... ok
test run_file_no_tests ... ok

test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished 


running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished 

